src/app/routes/index.js

```diff
@@ -2,7 +2,7 @@ import article from './article';
 import cpsAsset from './cpsAsset';
 import home from './home';
 import radio from './radio';
-import error200 from './error200';
-import error404 from './error404';
+import error from './error';
+import notFound404 from './404';
```

Comment:
Can you please rename the 404 page as errorNoRouteMatch. It ensures it's colocated with the error page and it has a distinction between the purpose of this one as opposed to `error`.
```suggestion
import notFound404 from './errorNoRouteMatch';
```